build.xml

```diff
@@ -439,18 +466,44 @@
       <istrue value="${seleniumjar}" />
       <then>
         <exec command="pkill -f xvfb" />
-        <exec command="pkill -f &quot;java -jar `basename ${seleniumjar}`&quot;" />
+        <basename file="${seleniumjar}" property="seleniumbase" />
+        <exec command="pkill -f &quot;java -jar ${seleniumbase}&quot;" />
       </then>
     </if>
 
     <!-- delete the configuration, sample index, logs and cache data -->
-    <delete file="${srcdir}/import/solrmarc.log" failonerror="true" />
-    <delete dir="${srcdir}/solr" includeemptydirs="true" failonerror="true" />
-    <delete file="${srcdir}/composer.phar" failonerror="false" />
-    <delete file="${srcdir}/vufind-exception.log" failonerror="false" />
+    <if>
+      <available file="${srcdir}/import/solrmarc.log" />
+      <then>
+        <delete file="${srcdir}/import/solrmarc.log" failonerror="true" />
+      </then>
+    </if>
+    <if>
+      <available file="${srcdir}/solr" type="dir" />
+      <then>
+        <delete dir="${srcdir}/solr" includeemptydirs="true" failonerror="true" />
+      </then>
+    </if>
+    <if>
+      <available file="${srcdir}/composer.phar" />
+      <then>
+        <delete file="${srcdir}/composer.phar" failonerror="false" />
```

Comment:
Since it's inside the available check, we could probably add failonerror="true" here too...